src/services/data/cmsDataQueryGateway.ts

```diff
@@ -91,7 +91,9 @@ export async function fetchAPIGatewayWrapper(details: DynamicFileModuleDetails,
 
         return await fetchAPI(details, endpoint, headers, pageAndSingleComponentDetails);
     } else if(cmsVariant === 'sanity') {
-      const endpoint = `${CmsVariants.variants[cmsVariant].deliveryApiDomain}`;
+      const projectId = process.env.SANITY_STUDIO_PROJECT_ID || "5he8nsc5";
```

Comment:
In the starter kit    SANITY_STUDIO_PROJECT_ID  is not defined this is a sanity studio env only. 

In the starter kit project the following generic is use: 

PROJECT_ID